src/web/EmployerFavouritesWeb/wwwroot/images/favourite-unchecked-black.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this be part of the das-frontend project rather than here?

Reply:
@sdholder do you have a response to Matt's comment?

Reply:
According to Steve the is a conscious decision for application specific assets and styles to remain in the app instead of going into das-frontend.